File: 1598-crawler-log-folder/1598-crawler-log-folder.java
class Solution {
    public int minOperations(String[] logs) {
        int ans=0;
        Stack<String> st=new Stack<>();
        for(String log:logs){
            if(log.equals("../")){
                if(!st.isEmpty()){
                    st.pop();
                }
            }else if(!log.equals("./")){
                st.push(log);
            }
        }
        System.out.println(st);
        return st.size();
    }
}

File: 1598-crawler-log-folder/1598-crawler-log-folder.py
class Solution(object):
    def minOperations(self, logs):
        """
        :type logs: List[str]
        :rtype: int
        """
        count=0
        for i in range(len(logs)):
            if '../' in logs[i]:
                count-=1
            elif './' in logs[i]:
                continue
            else:
                count+=1
            if count<0:
                count=0
        return count
        